### ISSUE TITLE
Find a cool domain name

### ISSUE THREAD

**Issue by swapagarwal** [swapagarwal/awesome-eli5#2]
Something like `eli5.tech` can be redirected to this page. Thoughts?

**Comment by Gerrymandered** [comment]
On Namecheap, eli5.tech is much more expensive than eli5.technology.

However, eli5.link is the best option I found. (Edit: $8.88/year)

**Comment by Jen-Ffm** [comment]
treat-me-like-i'm 5.com

**Comment by serenevoid** [comment]
eli5.top is available for $0.99 in epik.com. Try that
